IndustrialUnit.WpfUI/ViewModels/EquipmentViewModel.cs

```diff
@@ -0,0 +1,198 @@
+﻿using IndustrialUnit.WpfUI.Models;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Data;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Controls.Primitives;
+using System.Windows.Input;
+
+namespace IndustrialUnit.WpfUI.ViewModels
+{
+  public class EquipmentViewModel : BaseViewModel
```

Comment:
Keveredik a grid (egész tábla) és a kiválasztott elemhez tartozó viewmodel.
DataView helyett lehetne egy ObservableCollection<Equipment>, a text boxokat pedig a DataGrid.SelectedItem-hez is lehet bindolni.